app/services/audit_results.rb

```diff
@@ -115,15 +115,25 @@ def remove_db_updated_results
 
   def report_results
     # Report completed
-    completed_results = result_array.select { |result| status_changed_to_ok?(result) }
```

Comment:
Extract variable to (public) method. This lets collaborators act on error results without needing knowledge of how to discern error results from non-error results.

Reply:
as discussed the other day on slack: reminder to @jmartin-sul to file a ticket to:
* add a list of non-error result codes
* and to switch to using that list for determining whether a result is "ok", instead of `status_changed_to_ok?`'s current regexp logic (which i think will allow some non-error result codes to be classified as errors, though i also don't think that'll be an issue for the replication result codes we care about in this PR, hence dealing with it as follow on work)

Reply:
Sounds good!